app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -44,6 +47,10 @@ def notify(op, manager_id, params, success)
       )
     end
 
+    def notify_refresh_in_provider(manager_id, params, success)
+      notify('refresh in provider', manager_id, params, success)
```

Comment:
It would definitely work like that. However I decided to make it a method, so the `'refresh in provider'` string describing the notification is only in one place. It’s like the `notify('creation', …)`, but this `'refresh in provider'` is called from two places: after create and after update.

Reply:
Okay, I’ll remove the method and call `notify` directly. I just liked the method more than calling `notify` with the same string from two different places. To me it conveyed a message saying “this is the same thing” making clear it is an intention. The string argument on the other hand is arbitrary and could be the same only by accident.

Reply:
Okay, I’ll remove the method and call `notify` directly. I just liked the method more than calling `notify` with the same string from two different places. To me it conveyed a message saying “this is the same thing” making clear it is an intention. The string argument on the other hand is arbitrary and could be the same only by accident.